recipes/cmsis-pack-manager/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "cmsis-pack-manager" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cmsis-pack-manager-{{ version }}.tar.gz
+  sha256: 35e506e8f148d9e4f0ab948db4007a64c035337cf526630d0f6f55f7f3b9b20c
+
+build:
+  entry_points:
+    - pack-manager = cmsis_pack_manager.pack_manager:main
+  script: {{ PYTHON }} -m pip install . -vv 
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+  host:
+    - python
+    - pip
+  run:
+    - appdirs >=1.4,<2.0
+    - milksnake >=0.1.5,<1.0
+    - python
+    - pyyaml >=6.0,<7.0
+
+test:
+  commands:
+    - pack-manager --help
+
+about:
+  home: https://github.com/pyocd/cmsis-pack-manager
+  summary: Python manager for CMSIS-Pack index and cache with fast Rust backend
+  license: Apache-2.0
+  license_file: LICENSE
```

```suggestion
  license_file:
    - LICENSE
    - add-bundled-licenses-file-here
```
You need to add the license file created by cargo-bundle-licenses here, and I'm pretty sure you need to run the cargo-bundle-licenses in a script. It doesn't run itself just because you added it to the build requirements.